README.md

```diff
@@ -21,8 +21,10 @@ composer require --dev jonmldr/grumphp-doctrine-task
 # grumphp.yml
 parameters:
     tasks:
-        doctrine_schema_validate: ~
-
+        doctrine_schema_validate:
+            skip_mapping: false
+            skip_sync: false
```

Comment:
The `skip_sync` option can be `true` by default in my opinion, the state of the local database should not prevent committing.

Reply:
db sync is validated by default in [the original grumphp task](https://github.com/phpro/grumphp/blob/master/doc/tasks/doctrine_orm.md), I would prefer it works exactly the same as that one, so migration would be frictionless.

It is also how we intend to use it, as we started looking to add this task because a PR was missing a migration, which would only be caught if db sync is also checked.

Reply:
I see your point now, thanks for your explanation! I will merge this PR and release version 2.0 (major GrumPHP dependency update)

Reply:
perfect, thanks!